DESCRIPTION

```diff
@@ -8,7 +8,9 @@ Description: Provides an Rmd macro to insert
 Depends:
     R (>= 3.3.0)
 Imports:
-    gtools
+    gtools,
+    RCurl,
+    utils
```

Comment:
I don't think I did a full CRAN-ready check (I have doubts this would make it to CRAN, but it *should* pass the tests so that others can install it) so these were very likely overlooked. Thank you for fixing this, it's greatly appreciated. I'm guessing at least `devtools` should also be there. I'll test and merge this and run a full check to see what else was missing.